### PR TITLE
[Spring] modify help text of creating service and app

### PR DESCRIPTION
**Pull request by RuoyuWang-MS**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repository and upgrade the version in the pull request but do not modify `src/index.json`.

### CODE REVIEW

**Comment by RuoyuWang-MS** [comment]
[CLI extension: create --help: --name argument: should say "Name of the Azure Spring Cloud Service instance".](https://github.com/Azure/Azure-Spring-Apps/issues/8) mentioned the help text issue and this PR fixes this issue.

**Comment by yonzhan** [comment]
Spring

File: src/spring/azext_spring/_params.py [hunk 1]
```diff
@@ -217,7 +217,7 @@ def load_arguments(self, _):
 
     with self.argument_context('spring app') as c:
         c.argument('service', service_name_type)
-        c.argument('name', name_type, help='Name of app.')
+        c.argument('name', name_type, help='The name of app running within the specified Azure Spring Apps instance.')
```
Comment (allxiao) [inline src/spring/azext_spring/_params.py:220]: Will `in` better than `within` here? (and also in below contexts)

**Comment by allxiao** [comment]
lgtm

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**